### PR TITLE
T12921 remove session from di factory

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: new feature
* Link to issue: #12921 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Removed the `session` service from the `FactoryDefault` container

Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay

**Review by sergeyklay — CHANGES_REQUESTED** [review]
There was 1 failure:
```
Phalcon\Test\Unit\Di\FactoryDefault\ConstructCest::diFactoryDefaultConstructServices
```

> Failed asserting that 19 matches expected 21

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -59,6 +59,7 @@
 - Scope SQL Column Aliases (on nesting level), in `Phalcon\Mvc\Model\Query`, to prevent overwrite _root_ query's `_sqlColumnAliases` by sub-queries. [#13006](https://github.com/phalcon/cphalcon/issues/13006), [#12548](https://github.com/phalcon/cphalcon/issues/12548) and [#1731](https://github.com/phalcon/cphalcon/issues/1731)
 - CLI parameters now work like MVC parameters [#12375](https://github.com/phalcon/cphalcon/pull/12375)
 - Changed `Phalcon\Db\Dialect\Postgresql::addPrimaryKey` to make primary key contraints names unique by prefixing them with the table name. [#12629](https://github.com/phalcon/cphalcon/pull/12629)
+- Changed `Phalcon\DI\FactoryDefault` to not load the `Phalcon\Session\Adapter\Files` using the name `session` by default [#12921](https://github.com/phalcon/cphalcon/issues/12921)
```
Comment (sergeyklay) [inline CHANGELOG-4.0.md]:
```suggestion
- Changed `Phalcon\DI\FactoryDefault` to not load by default
    - the `Phalcon\Session\Adapter\Files` using the name `session` 
    - the `Phalcon\Session\Bag` using the name `sessionBag` 
  [#12921](https://github.com/phalcon/cphalcon/issues/12921)
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
@sergeyklay fixed issues

**Review by sergeyklay — APPROVED** [review]

**Pull request merged**